src/desktops/dskSelectMap.cpp

```diff
@@ -91,6 +109,10 @@ dskSelectMap::dskSelectMap(const CreateServerInfo& csi)
     AddTextButton(4, 590, 530, 200, 22, TC_GREEN2, _("Load game..."), NormalFont);
     // "Weiter"
     AddTextButton(5, 590, 560, 200, 22, TC_GREEN2, _("Continue"), NormalFont);
+    // random map generation
+    AddTextButton(6, 380, 530, 150, 22, TC_GREEN2, _("Random"), NormalFont);
+    // random map players
+    AddTextButton(7, 540, 530, 40, 22, TC_GREEN2, _("..."), NormalFont);
```

Comment:
My thought behind this was that I want to click only once for regenerating the map. If I always open the settings it takes longer to find the map I like to play. Therefore, I split up the generation and the settings into two separate buttons. Just give it a try, you'll see it is more comfortable this way ;)